sentry-android-fragment/src/main/java/io/sentry/android/fragment/SentryFragmentLifecycleCallbacks.kt

```diff
@@ -71,13 +92,18 @@ class SentryFragmentLifecycleCallbacks(
 
     override fun onFragmentDestroyed(fragmentManager: FragmentManager, fragment: Fragment) {
         addBreadcrumb(fragment, "destroyed")
+
+        stopTracing(fragment)
```

Comment:
`h`: On Cocoa, we finish on `onResume`. The duration of the fragment transaction would vary greatly.

Reply:
we also finish on `onFragmentResumed`, this is more like a safe guard to not leak anything when the fragment gets destroyed